src/types/index.ts

```diff
@@ -158,6 +158,7 @@ export interface ITx {
   returnType: string,
   type: string,
   callerId: string,
+  accountId?: string,
```

Comment:
This really exists in the API response data? If not I suggest to create separate interface that would extend this one to distinguish raw data form data that we extend with own logic.

Reply:
Yes it does